bindata/v4.0.0/configmap-cabundle-controller/role.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
```

Comment:
I believe these are all the same and cover the namespace scoped library-go functionality.  We should probably just share a single role here.  A single role binding with all three controllers would also make sense.

Reply:
I was weighing the trade off between less moving parts with sharing a role/rolebinding, and maintaining the independence of each controller. Like, right now you can delete any of the controllers and it's respective resources and the other run just fine; or you can run them independently with its set of resources. WDYT?

Reply:
I think the controllers can be coupled together if that makes deployments simpler, there's no situation where you would want to only run a subset of them.